src/Game.js

```diff
@@ -40,6 +42,50 @@ class Game extends Component {
     return spritesPos;
   }
 
+  /* 
```

Comment:
Can you remove these comments. There are various ways on GitHub to know who contributed what code. There's Contributions graphs, `git blame`.

Plus I'll be adding every contributors name in the README.

Comments like these obfuscate the actual logic.

Reply:
Yes, you're absolutely right. I will remove all the unnecessary comments.